packages/strapi-admin/admin/src/validations/users/profile.js

```diff
@@ -10,10 +10,10 @@ const schema = {
     .email(translatedErrors.email)
     .lowercase()
     .required(translatedErrors.required)
-    .test('topLevelDomainValidation', translatedErrors.email, function(value) {
-      let parseDomainResult = parseDomain(value);
-      
-return parseDomainResult.type === ParseResultType.Listed;
+    .test('topLevelDomainValidation', 'Settings.users.validations.email-domain', function(value) {
```

Comment:
I made the fix but I believe that this is not a wanted logic we want in the admin panel. IMO, this is something that the users should add in their app if they want such validation.

WDYT @alexandrebodin ?

Reply:
Not sure what was the wanted behaviour. We can revert it as this is still doable in a lifecycle hook

Reply:
I think it would be better to revert it yes

Reply:
you can go ahead